ui/ducks/send/send.js

```diff
@@ -2190,12 +2190,14 @@ export function useMyAccountsForRecipientSearch() {
  * @returns {ThunkAction<void>}
  */
 export function resetRecipientInput() {
-  return async (dispatch) => {
+  return async (dispatch, getState) => {
+    const state = getState();
+    const chainId = getCurrentChainId(state);
     await dispatch(addHistoryEntry(`sendFlow - user cleared recipient input`));
     await dispatch(updateRecipientUserInput(''));
     await dispatch(updateRecipient({ address: '', nickname: '' }));
     await dispatch(resetEnsResolution());
-    await dispatch(validateRecipientUserInput());
+    await dispatch(validateRecipientUserInput({ chainId }));
```

Comment:
I looked at the implementation of validateRecipientUserInput [here](https://github.com/MetaMask/metamask-extension/blob/0cddaa9098d17dd161fafcea53699228e45fcac9/ui/ducks/send/send.js#L1339) and I'm not sure how the chainId is used in that definition.

Reply:
Its used to determine if the current chain is one of the default metamask chains, which changes what error message is shown. I think its used one more time in that definition but thats the first usage making it required.